docs/products/kafka/kafka-connect/howto/debezium-source-connector-sql-server.rst

```diff
@@ -0,0 +1,193 @@
+Create a Debezium source connector for SQL Server
+==================================================
+
+The SQL Server Debezium source connector is based on the `change data capture feature <https://docs.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-data-capture-sql-server?view=sql-server-2017>`, extracts the database changes captured in specific `change tables <https://debezium.io/documentation/reference/stable/connectors/sqlserver.html>`_ on a polling interval, and writes them to an Apache Kafka® topic in a standard format where they can be transformed and read by multiple consumers.
```

Comment:
I'd split the long sentence into two, like this - 
```suggestion
The SQL Server Debezium source connector is based on the `change data capture feature <https://docs.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-data-capture-sql-server?view=sql-server-2017>`_. The connector extracts the database changes that are captured in a specific `change table <https://debezium.io/documentation/reference/stable/connectors/sqlserver.html>`_, and writes them to an Apache Kafka® topic in a standard format where they can be transformed and read by multiple consumers. The extraction happens on a polling interval.
```

I also rephrased it a bit to add more short sentences. "on a polling interval" sounds a bit weird to me, but I can't find a synonym.